week10_planning/seminar_MCTS.ipynb

```diff
@@ -6,46 +6,58 @@
    "source": [
     "## Seminar: Monte-carlo tree search (5 pts)\n",
     "\n",
-    "In this seminar, we'll implement a vanilla MCTS planning and use it to solve some Gym envs.\n",
+    "Monte Carlo tree search (MCTS) is a heuristic search algorithm, which shows cool results in challenging domains such as Go and chess. The algorithm involves building a search tree, iteratively traversion  and evaluation the nodes with Monte-Carlo simulation.\n",
     "\n",
-    "![image.png](https://i.postimg.cc/6QmwnjPS/image.png)\n"
+    "In this seminar, we'll implement a MCTS([[1]](#1), [[2]](#2)) planning and use it to solve some Gym envs.\n",
+    "\n",
+    "![image.png](https://i.postimg.cc/6QmwnjPS/image.png)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "__How it works?__\n",
-    "\n",
-    "We just start with an empty tree and from there we iterate and expand it. There are several common procedures.\n",
+    "We just start with an empty tree and expand it. There are several common procedures.\n",
     "\n",
     "__1) Selection__\n",
+    "Starting from the root, recursively select the node that corresponds tree policy.  \n",
     "\n",
-    "Starting from the root, recursively select node with highest UCB-1 score - on the one hand, the most beneficial action, on the other hand, the least explored.\n",
+    "Widely used _tree policy_ - choose the action with the highest UCB-1 score. On one hand, this is the most beneficial action; on the other hand, it's the least explored one.\n",
```

Comment:
I don't understand what `Widely used _tree policy_` is supposed to mean here.

Reply:
I mean, that, there are several tree policies, which we saw earlier as exploration strategies: epsilon-greedy, Thomson sampling, UCB-1. But was shown that in MCTS good results can achieve with UCB-1. And further, we will consider the one, but you can try to use others.